uiowa/uiowa
# [Facilities] Projects content type

Resolves #6398 





# How to test


`ddev blt ds --site=facilities.uiowa.edu`
As administrator, create some Project content, test the fields function as they should

- Project Image - single image media field
- What's New - text area currently filtered html
- Project Description - text area currently filtered html
- Webcam - link field - title + url - title defaults to "View webcam"
- SEO Settings setup similar to other content types
- Schema.org coverage as https://schema.org/OrganizeAction which I thought suited this content type better than https://schema.org/Project

Masquerade as other roles. They should be able to edit existing project and change the above fields, but they should not be able to create new projects or delete existing projects

Check that the general bases have been covered for creating a new content type (eg https://wiki.uiowa.edu/display/drupaldev/Content+type+creation+notes)
